--- feeds.conf.default	2021-11-23 01:54:21.997450324 +0000
+++ feeds.conf.default	2021-11-23 01:59:31.226026854 +0000
@@ -6,5 +6,6 @@
 #src-git targets https://github.com/openwrt/targets.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
-src-git lienol https://github.com/Lienol/openwrt-package.git;main
+#src-git lienol https://github.com/Lienol/openwrt-package.git;main
 src-git other https://github.com/Lienol/openwrt-package.git;other
+src-git passwall https://github.com/alecthw/openwrt-passwall.git;main
