--- feeds.conf.default	2021-08-17 16:52:56.581440060 +0800
+++ feeds.conf.default	2021-08-17 16:54:01.773436891 +0800
@@ -6,4 +6,6 @@
 #src-git targets https://github.com/openwrt/targets.git
 #src-git oldpackages http://git.openwrt.org/packages.git
 #src-link custom /usr/src/openwrt/custom-feed
-src-git lienol https://github.com/Lienol/openwrt-package.git;main
+#src-git lienol https://github.com/Lienol/openwrt-package.git;main
+src-git passwall https://github.com/xiaorouji/openwrt-passwall.git;main
+src-git n2n https://github.com/alecthw/openwrt-n2n.git;master
